index.html

```diff
@@ -338,7 +338,62 @@
 	<li style="background-color: gray;
 background-image: linear-gradient(transparent 50%, rgba(255,255,255,.5) 50%);
 background-size: 50px 50px;" title="Horizontal stripes"></li>
-	
+
+
+<li style="background:
+radial-gradient(circle closest-side,#FC4 40%,rgba(240,166,17,0) 90%)20px 30px,
+radial-gradient(circle closest-side,#FC4 40%,rgba(240,166,17,0) 90%)0 0,
+radial-gradient(circle farthest-side at 0% 50%,#fb1 22.8%,rgba(240,166,17,0) 23.8%)21px 30px,
+radial-gradient(circle farthest-side at 0% 50%,#B71 24%,rgba(240,166,17,0) 25%)19px 30px,
+linear-gradient(#fb1 15%,rgba(240,166,17,0) 16%, rgba(240,166,17,0) 84%,#fb1 85%)0 0,
+linear-gradient(150deg,#fb1 23.5%,#B71 24%,#B71 26%,rgba(240,166,17,0) 26.5%,rgba(240,166,17,0) 73.5%,#B71 74%,#B71 76%,#fb1 76.5%)0 0,
+linear-gradient(30deg,#fb1 23.5%,#B71 24%,#B71 26%,rgba(240,166,17,0) 26.5%,rgba(240,166,17,0) 73.5%,#B71 74%,#B71 76%,#fb1 76.5%)0 0,
+linear-gradient(90deg,#B71 2%,#fb1 2.5%,#fb1 97.5%,#B71 98%)0 0 #fb1;
+background-size:40px 60px;"
+title="HoneyComb" data-author="Paul Salentiny"></li>
+
+<li style="
+background-color:steelblue; 
+background-image: 
+linear-gradient(#ffffff 50%, rgba(255,255,255,0) 50%),
+radial-gradient(circle closest-side, #FFFFFF 53%, rgba(255,255,255,0) 54%),
+radial-gradient(circle closest-side, #FFFFFF 50%, rgba(255,255,255,0) 51%);
+background-size:110px 200px;background-position: 0 0, 0 0, 55px 0;
+background-repeat:repeat-x;"
+title="Wave" data-author="Paul Salentiny"></li>
+
+<li style="background-color: lightsteelblue;
+background-image: linear-gradient(225deg, transparent 94%, #3c5f49 94%), 
+linear-gradient(135deg, transparent 94%, #3c5f49 94%), 
+linear-gradient(225deg, transparent 91%, #3c5f49 91%), 
+linear-gradient(135deg, transparent 91%, #3c5f49 91%), 
+linear-gradient(225deg, transparent 88%, #3c5f49 88%), 
+linear-gradient(135deg, transparent 88%, #3c5f49 88%), 
+linear-gradient(225deg, transparent 87%, #3c5f49 87%), 
+linear-gradient(135deg, transparent 87%, #3c5f49 87%), 
+linear-gradient(262deg, transparent 92%, #4b3b2a 92%), 
+linear-gradient(99deg, transparent 92%, #4b3b2a 92%);
+background-size: 60px 100px;
+background-position: -100px -40px, -100px -40px, -100px -30px, -100px -30px, -100px -20px, -100px -20px, -100px -10px, -100px -10px, -102px 0, -98px 0;"
+title="Tree" data-author="Paul Salentiny"></li>	
+
+<li style="background-color: #000;
+background-image: linear-gradient(315deg, transparent 75%, #d45d55 75%), 
+linear-gradient(45deg, transparent 75%, #d45d55 75%), 
+linear-gradient(135deg, #a7332b 50%, transparent 50%), 
+linear-gradient(45deg, #6a201b 50%, transparent 50%), 
+linear-gradient(45deg, transparent 50%, #561a16 50%);
+background-size: 20px 20px;
+background-position: -10px 0, -10px 0, 0 0, 0 0, 0 0;"
+title="Pyramid" data-author="Paul Salentiny"></li>
```

Comment:
I wonder if we could set a base color and specify the rest as semi-transparent black.
Also, I’m pretty sure the number of gradients can be reduced, but don’t have enough time to do it myself right now.